source/01-LE/01.ptx

```diff
@@ -520,6 +568,15 @@ All inconsistent linear systems contain a logical <term>contradiction</term>.
       a_{m1} &amp; a_{m2} &amp; \cdots &amp; a_{mn} &amp; b_m
     \end{array}\right]
             </me>
+            Often, we will want to refer only to the coefficients of the linear system; the <term>coefficient matrix</term><idx>coeefficient matrix</idx> is the <m>m\times n</m> array consisting of these coefficients:
```

Comment:
```suggestion
            Eventually, we will find it useful to refer only to the coefficients of the linear system
            (and ignore its constant terms).
            We call the <m>m\times n</m> array consisting of these coefficients 
            a <term>coefficient matrix</term><idx>coeefficient matrix</idx>.
```

Reply:
hmmm...I think I'd prefer "Sometimes" to "Eventually", as minor as it may seem.

Reply:
i've updated my suggestion accordingly